### ISSUE TITLE
 Cannot read property '_cast' of undefined after update to 0.10.5

### ISSUE THREAD

**Issue by JumpLink** [balderdashy/sails-disk#33]
After update my sails project from sails 0.10.0 rc9 to 0.10.5 (sails-disk is also updated to 0.10.5) I get an error when I try to start the application. After downgrade sails-disk to 0.10.4 it is working.

The Error on sails-disk 0.10.5:
```
% sails lift                                                                         :(

info: Starting app...

/[myProject]/node_modules/sails-disk/lib/database.js:209
      self.collections[collectionName]._cast.run(values);
                                      ^
TypeError: Cannot read property '_cast' of undefined
    at /[myProject]/node_modules/sails-disk/lib/database.js:209:39
    at Array.map (native)
    at Database.read (/[myProject]/node_modules/sails-disk/lib/database.js:208:32)
    at Array.async.auto.checkData [as 0] (/[myProject]/node_modules/sails-disk/lib/database.js:59:12)
    at /[myProject]/node_modules/sails-disk/node_modules/async/lib/async.js:459:38
    at Array.forEach (native)
    at _each (/[myProject]/node_modules/sails-disk/node_modules/async/lib/async.js:32:24)
    at Object.async.auto (/[myProject]/node_modules/sails-disk/node_modules/async/lib/async.js:430:9)
    at Database.initialize (/[myProject]/node_modules/sails-disk/lib/database.js:55:9)
    at Object.module.exports.adapter.registerConnection (/[myProject]/node_modules/sails-disk/lib/adapter.js:51:40)
```

**Comment by articice** [comment]
I had the same error, when I decided to move one of my models to a different adapter/connection. The sails-disk adapter was too slow, so I had to move to Redis, even for development purposes.
I manually deleted all the relevant data from .tmp/localDiskDb (a couple thousands of records for this particular model) and it started working.

**Comment by sailsbot** [comment]
Thanks for posting, @JumpLink.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
